tests/integration/backup/simple/utils.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2022 Democratized Data Foundation
```

Comment:
todo: Fix copyright year.

```suggestion
// Copyright 2023 Democratized Data Foundation
```